### PR TITLE
[ozone] adding warnings for illegal volume and bucket name

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- adding warnings for illegal volume and bucket name
- allowed length 3-63
- upper case not allowed

## How was this patch tested?

- locally

### CODE REVIEW

File: apps/filebrowser/src/filebrowser/templates/listdir_components.mako [hunk 1]
```diff
@@ -1854,6 +1860,18 @@ else:
           resetPrimaryButtonsStatus(); //globally available
           return false;
         }
+        if ($("#newDirectoryNameInput").val().length < 3 && (self.isOFSRoot() || self.isOFSVol())) {
```
Comment (JohanAhlen) [inline apps/filebrowser/src/filebrowser/templates/listdir_components.mako:1863]: Max length check missing?

`... || $("#newDirectoryNameInput").val().length > 63) && (self.isOFSRoot() ...`

**Review by JohanAhlen — COMMENTED** [review]
Nice!

File: apps/filebrowser/src/filebrowser/templates/listdir_components.mako [hunk 2]
```diff
@@ -1854,6 +1860,18 @@ else:
...
+          $("#volumeBucketNameAlert").show();
+          $("#newDirectoryNameInput").addClass("fieldError");
+          resetPrimaryButtonsStatus(); //globally available
+          return false;
+        }
+        if (($("#newDirectoryNameInput").val() !== $("#newDirectoryNameInput").val().toLowerCase()) && (self.isOFSRoot() || self.isOFSVol())) {
```
Comment (JohanAhlen) [inline apps/filebrowser/src/filebrowser/templates/listdir_components.mako:1869]: nit: You could consider surrounding both checks with one if for `self.isOFSRoot() || self.isOFSVol()`

File: apps/filebrowser/src/filebrowser/templates/listdir_components.mako [hunk 3]
```diff
@@ -501,6 +501,12 @@ else:
         <div id="smallFileSystemNameAlert" class="hide" style="position: absolute; left: 10px;">
           <span class="label label-important"><span class="newName"></span> ${_('File system requires namesize to be 3 or more characters')}</span>
         </div>
+        <div id="volumeBucketNameAlert" class="hide" style="position: absolute; left: 10px;">
+          <span class="label label-important"><span class="newName"></span> ${_('Volume and Bucket requires namesize to be 3-63 characters')}</span>
```
[the remark this answers is not in the document]
Comment (agl29) [reply apps/filebrowser/src/filebrowser/templates/listdir_components.mako]: Done

File: apps/filebrowser/src/filebrowser/templates/listdir_components.mako (hunk 1 above)
Comment (agl29) [reply apps/filebrowser/src/filebrowser/templates/listdir_components.mako]: Thanks, Done

**Review by bjornalm — APPROVED** [review]
Nice work

**Review by JohanAhlen — APPROVED** [review]
LGTM!

**Pull request merged**